sphinx/writers/html.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, builder, *args, **kwds):
         self.highlightopts = builder.config.highlight_options
         self.highlightlinenothreshold = sys.maxsize
         self.docnames = [builder.current_docname]  # for singlehtml builder
+        self.manpages_url_pattern = getattr(builder.config, 'manpages_url_pattern')
```

Comment:
After adding an entry to `sphinx.config`, you can access it always. So you don't need to use `getattr` here. 
BTW, why are you storing the value of `manpage_url_pattern` as an instance variable?

Reply:
because that's the pattern used for other configuration, like `highlightopts`.

Reply:
and the reason why i'm using `getattr` is so that older configuration files keep working, but maybe that's not necessary?

Reply:
Yes, it's not necessary. It is always enabled even if with old config-files, because you'd added it to `sphinx.config`.

>because that's the pattern used for other configuration, like highlightopts.

I see, either is okay to me.

Reply:
okay, fixed.